app/src/main/java/org/mozilla/fenix/settings/sitepermissions/SitePermissionsManagePhoneFeatureFragment.kt

```diff
@@ -213,6 +214,9 @@ class SitePermissionsManagePhoneFeatureFragment : Fragment() {
         requireComponents.analytics.metrics.track(Event.AutoPlaySettingChanged(setting))
         settings.setSitePermissionsPhoneFeatureAction(AUTOPLAY_AUDIBLE, audible)
         settings.setSitePermissionsPhoneFeatureAction(AUTOPLAY_INAUDIBLE, inaudible)
+        context?.let {
+            it.components.useCases.sessionUseCases.reload()
+        }
```

Comment:
The usage of `let` seems unnecessary here?

```Kotlin
context?.components?.useCases?.sessionUseCases?.reload()
```

Reply:
Yeah, a bit  I wasn't sure if all the `?` were OK :sweat_smile:, I will update it.

Reply:
We additionally need `reload?.invoke()` 😅

Reply:
Fixed!